utilities/src/main/java/io/syndesis/qe/utils/OpenShiftUtils.java

```diff
@@ -65,20 +64,45 @@ public static OpenShiftBinary binary() {
 
     private OpenShiftUtils() {
         if (xtfUtils == null) {
-            final OpenShiftConfigBuilder openShiftConfigBuilder = new OpenShiftConfigBuilder()
-                    .withMasterUrl(TestConfiguration.openShiftUrl())
-                    .withTrustCerts(true)
-                    .withRequestTimeout(120_000)
-                    .withNamespace(TestConfiguration.openShiftNamespace());
-            if (!TestConfiguration.openShiftToken().isEmpty()) {
-                //if token is provided, lets use it
-                //otherwise f8 client should be able to leverage ~/.kube/config or mounted secrets
-                openShiftConfigBuilder.withOauthToken(TestConfiguration.openShiftToken());
-            }
-            xtfUtils = new OpenShift(openShiftConfigBuilder.build());
+            useAdminUser();
```

Comment:
So since we are already touching this code, maybe we could get rid of the "initialize static field in private constructor that is called from the singleton access method" maze of calls and just initialize the `xtfUtils` in `getInstance`?